src/symbols/Node/Node.tsx

```diff
@@ -147,24 +143,24 @@ export const Node: FC<Partial<NodeProps>> = ({
   onDrag = () => undefined,
   onDragStart = () => undefined,
   onDragEnd = () => undefined,
-  onClick = null,
-  onKeyDown = null,
-  onEnter = null,
-  onLeave = null
+  onClick = () => undefined,
```

Comment:
Wouldn't it be more ideal to leave these null or undefined and then do something like `onClick.?()`?

Reply:
done